version: "3.9"

services:
  homeassistant:
    container_name: homeassistant
    logging:
      driver: "json-file"
      options:
        max-size: 256m
        max-file: "1"
    image: homeassistant/home-assistant
    environment:
      TZ: "America/Los_Angeles"
    volumes:
       - "/zpool1/docker/homeassistant/config:/config:rw"
    restart: always
    labels:
      - "autoheal=false"
      - "traefik.enable=true"
      - "traefik.http.routers.homeassistant.entrypoints=web,websecure"
      - "traefik.http.routers.homeassistant.rule=Host(`homeassistant.shazbot.one`)"
      - "trafeik.http.routers.homeassistant.tls.certresolver=letsencrypt"
      - "traefik.http.routers.homeassistant.tls.domains[0].main=shazbot.one"
      - "traefik.http.routers.homeassistant.tls.domains[0].sans=*.shazbot.one"
      - "traefik.http.services.homeassistant.loadbalancer.server.port=8123"
    deploy:
     resources:
       limits:
         memory: 1g
    healthcheck:
      test: "curl localhost:8123"
      interval: "60s"
      timeout: "3s"
      start_period: "25s"
      retries: 3
    device_cgroup_rules:
      - 'c 188:* rmw'
